python/pythonmonkey/builtin_modules/console.js

```diff
@@ -49,8 +47,15 @@ class Console {
         ignoreErrors = true;
       options = { stdout, stderr, ignoreErrors };
     }
+
     this.#writeToStdout = options.stdout.write;
     this.#writeToStderr = options.stderr.write;
+
+    this.log   = (...args) => this.#writeToStdout(this.#formatToStr(...args));
+    this.debug = (...args) => this.#writeToStdout(this.#formatToStr(...args));
+    this.info  = (...args) => this.#writeToStdout(this.#formatToStr(...args));
+    this.warn  = (...args) => this.#writeToStdout(this.#formatToStr(...args));
+    this.error = (...args) => this.#writeToStdout(this.#formatToStr(...args));
```

Comment:
`console.warn` and `console.error` should output to stderr
```suggestion
    this.warn  = (...args) => this.#writeToStderr(this.#formatToStr(...args));
    this.error = (...args) => this.#writeToStderr(this.#formatToStr(...args));
```

Reply:
Oops! Good catch - cut and paste error.

I should add a test for this.